releasenotes/releasenotes_V0.7.1.md

```diff
@@ -0,0 +1,112 @@
+# Release Notes 0.7.1
+
+Keptn 0.7.1 improves the capabilities of the 0.7 release by adding more flexibility to the tasks of installing/operating Keptn, introducing two new CLI commands and improving the UX in the Keptn Bridge. Especially the new feature of installing Keptn in different namespaces is a door opener for various use-cases since multiple Keptn deployments, e.g., one for each team, can be operated on one cluster. 
+
+**The three key announcements of Keptn 0.7.1**:
+
+:rocket: *More flexibility in installing/operating Keptn*: 
+- `keptn install --namespace`: Allows you to specify the namespace to install Keptn in. 
+- `keptn upgrade`: Instead of manually executing a K8s job as done by the previous upgrade processes, this CLI command performs the upgrade. 
+- Set `root context`: As part of the installation Helm Chart of Keptn, a root context can be specified that allows you to customize the API endpoint of Keptn. 
+
+:star2: *New CLI commands*:
+- `keptn delete service`: This command will delete a service from a project and *undeploy* the service from the cluster. 
+- `keptn get events`: This command is a generic implementation to retrieve Keptn events of any event type.  
+
+:sparkles: *UX improvements in environment screen*: Based on feedback on the features of the Keptn delivery assistant, improvements regarding the user experience of the Keptn Bridge has been implemented.
+
+*Additional note:* Added documentation of [GOVERNANCE](https://github.com/keptn/keptn/blob/0.7.1/GOVERNANCE.md) & [SECURITY](https://github.com/keptn/keptn/blob/0.7.1/GOVERNANCE.md) process
+
+## Keptn Specification
+
+Implemented **Keptn spec** version: [0.1.5](https://github.com/keptn/spec/tree/0.1.5)
+
+## New Features
+
+<details><summary>Platform Support / Installer</summary>
```

Comment:
If we manage to run Keptn on OpenShift, we should add it here and in the highlights.

Reply:
agree! let's add it after a successful test.